commit 7352fc4a977a3414cf3194faf6ef990638a6329c
Author: Roman Yurchak <rth.yurchak@gmail.com>
Date:   Sat Feb 27 11:13:29 2021 +0100

    disable-blas-detection.patch

BLAS and LAPACK aren't available on host because we only cross compile these
libraries (see CLAPACK/meta.yaml). Scipy tries to detect these libraries and
if it fails to find them errors out the build. But we have them installed
correctly for our target, so we just disable this detection mechanism.

diff --git a/setup.py b/setup.py
index 0e11de60f..8c28c613d 100755
--- a/setup.py
+++ b/setup.py
@@ -484,7 +484,7 @@ def configuration(parent_package='', top_path=None):
 
     lapack_opt = get_info('lapack_opt')
 
-    if not lapack_opt:
+    if not lapack_opt and False:
         if sys.platform == "darwin":
             msg = ('No BLAS/LAPACK libraries found. '
                    'Note: Accelerate is no longer supported.')
